Style.css

```
@@ -1,43 +1,40 @@
 @import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');
+
 body {
-  margin: 0;
+  margin: 15px;
   font-size: 12px;
```

use relative font sizing.
```suggestion
  font-size: 0.8em;
```